bigbluebutton-html5/imports/ui/components/settings/component.jsx

```diff
@@ -217,6 +217,7 @@ class Settings extends Component {
         title={intl.formatMessage(intlMessages.SettingsLabel)}
         confirm={{
           callback: () => {
+            this.props.mountModal(null);
```

Comment:
The confirm button already has a function that hides the modal when it resolves the callback.
https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/modal/fullscreen/component.jsx#L43